discovery.mdx

```diff
@@ -6,14 +6,14 @@ icon: "microchip-AI"
 
 We are deploying an `AI agent` to discover user flows and interactions leading towards a desired user goal.
 It is the foundation for creating `deterministic test cases`. We are recording and storing the interaction chain of a test case in an intermediate representation.
-A Playwright code is generated on the fly just before test execution.
+Playwright code is generated on the fly just before test execution.
 
 ## Free prompting
 
-We have alpha released our free prompting interface for everyone to try it out. It will helps us fine tune the AI agent and the self-serve experience.
+We have alpha released our free prompting interface for everyone to try it out. It will help us fine tune the AI agent and the self-serve experience.
```

Comment:
```suggestion
We have released our free prompting interface for everyone to try it out as an alpha release. It will help us fine tune the AI agent and the self-serve experience.
```